src/plugins/mer/meremulatordevice.cpp

```diff
@@ -567,7 +494,7 @@ void MerEmulatorDevice::updateDconfDb(const QVariantMap &fullDeviceModelData)
 QVariantMap MerEmulatorDevice::readFullDeviceModelData() const
```

Comment:
The code from this function should be moved to MerSettings::read() as well. All the data for all device models should be read at once in MerSettings::read().

Reply:
Missed to remove this function. It is removed now.